test/tools/paths.d

```diff
@@ -60,7 +60,8 @@ string dmdModel()
         return dmdModel;
 
     const prefix = generatedDir.buildPath(os, build);
-    return dmdModel = prefix.buildPath("64", dmdFilename).exists ? "64" : "32";
+    return dmdModel = environment.get("DMD_MODEL",
+        prefix.buildPath("64", dmdFilename).exists ? "64" : "32");
```

Comment:
I'd rather prefer to override the full DMD path below instead.

Reply:
I went for the minimally invasive change here.
Are you okay with changing/improving this in a follow-up?

Reply:
Fine with me.